packages/components/nodes/chains/LLMChain/LLMChain.ts

```diff
@@ -135,10 +146,10 @@ const runPrediction = async (
             if (isStreaming) {
                 const handler = new CustomChainHandler(socketIO, socketIOClientId)
                 const res = await chain.call(options, [loggerHandler, handler, ...callbacks])
-                return res?.text
+                return applyOutputParser(res?.text, outputParser)
```

Comment:
we don't need to `parse` because its already been called inside LLMChain before returning the output `res` - https://js.langchain.com/docs/modules/model_io/output_parsers/how_to/use_with_llm_chain

Also, having this results in error: 
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/3204b0a1-3143-48aa-8480-edc854d16c6e)

Reply:
oh misunderstood your implementation, you're not using outputParser in the initialization of LLMChain
```
new LLMChain({
  llm: chatModel,
  prompt,
  outputKey: "records", // For readability - otherwise the chain output will default to a property named "text"
  outputParser: outputFixingParser,
})
```
is there any reason why not?

Reply:
I had some error (type mismatch) and need to spend more time investigating. Will dive deeper today. Ultimately, will need to get rid of this extra call.

Reply:
Done! I pushed a new commit.